airflow/www/utils.py

```diff
@@ -23,7 +23,7 @@
 standard_library.install_aliases()  # noqa: E402
 from builtins import str, object
 
-from cgi import escape
+from html import escape # python3.8 removed `escape` from `cgi` module
```

Comment:
This is fixed/not used in latest Airflow master. No worries we will later update Airflow to latest master.